published/external/xdpapi.h

```diff
@@ -474,6 +486,61 @@ XdpOpenApi(
     return _Status;
 }
 
+#define _POLL_INTERVAL_MS 10
```

Comment:
if this is only used in one function, i'd just make it a `static const` variable in that function